src/BudgetSharing/Expenso.BudgetSharing.Domain/BudgetPermissionRequests/Rules/ExpirationDateMustBeGreaterThanSubmissionDate.cs

```diff
@@ -15,7 +15,7 @@ public ExpirationDateMustBeGreaterThanSubmissionDate(DateAndTime submissionDate,
     }
 
     public string Message =>
-        $"Expiration date {_expirationDate.Value} must be greater than Submission date: {_submissionDate.Value}.";
+        $"Expiration date {_expirationDate} must be greater than Submission date: {_submissionDate}.";
```

Comment:
I'll remove that Print(format: "o") usage because it currently done by override ToString() method:

`    public override string ToString()
    {
        return Value.ToString(format: "o", formatProvider: CultureInfo.InvariantCulture);
    }`